include/ofi_mr.h

```diff
@@ -327,6 +327,30 @@ bool ofi_mr_cache_flush(struct ofi_mr_cache *cache, bool flush_lru);
 
 int ofi_mr_cache_search(struct ofi_mr_cache *cache, const struct fi_mr_attr *attr,
 			struct ofi_mr_entry **entry);
+
+enum ofi_mr_cache_status{
+	OFI_MR_CACHE_STATUS_NORMAL                = 0,
```

Comment:
No need to define a flag with the value 0, just use 0.  Developers used to using flags expect this behavior.

Reply:
I think this might also go back to the GDB discussion from earlier comments. If 0 instead of a label equal to 0, GDB won't parse the type and provide the label. It's a quality of life thing from my perspective. It doesn't cost us anything as far as I can tell.

Reply:
The general flow for checking flags is to do:
```
if (flags & FLAG_VALUE)
    do something because FLAG_VALUE is set
```
That does not work when FLAG_VALUE is 0.  FLAG_VALUE = 0 is not a flag -- it's the absence of all other flags being set.  That why you never see a flag defined as 0.  It's not usable.

An enum is an integer level data type.  However, flags are bits.  We can't apply a data type on a bit.

Reply:
I see the point here. It took a moment to think about it, but you cannot expect rational behavior with a 0-bit flag. Adding any other flag would 'remove' that 0-bit flag. I think what is present here is an odd blend between a return code and a flag.